files/zh-cn/web/css/css_fragmentation/index.md

```diff
@@ -1,16 +1,18 @@
 ---
-title: CSS 分片
+title: CSS 片段
 slug: Web/CSS/CSS_fragmentation
 ---
 
 {{CSSRef}}
 
-**CSS Fragmentation** 是 **CSS** 的模块，它定义了内容在跨多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)，区域或[列中](/zh-CN/docs/Web/CSS/CSS_multicol_layout)被分割（分段）时如何显示。
+**CSS 片段**模块定义了当内容在多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)、区域或[列](/zh-CN/docs/Web/CSS/CSS_multicol_layout)中被分割（fragmented）时的显示方式。
 
-当一个内联框包装成多行时会发生碎片。当一个块跨越一个列布局容器内的多个列，或者在打印时跨越一个分页符时，也会发生这种情况。元素的每个渲染*片段*称为一个*片段*。
+当行向盒子被分割到多行上时，就会出现分片。当一个块在列布局容器内跨越多列，或在打印时跨越分页符时，也会出现这种情况。元素的每个呈现片段都称为*分片*。
```

Comment:
The translation of "fragmentation" is not align with the context.

Reply:
```suggestion
当行向盒子被分割到多行上时，就会出现片段。当一个块在列布局容器内跨越多列，或在打印时跨越分页符时，也会出现这种情况。元素的每次呈现都称为*片段*。
```